src/Revise.jl

```diff
@@ -1040,6 +1040,9 @@ function method_location(method::Method)
     return method.file, method.line
 end
 
+# On Julia 1.5.0-DEV.260 and higher, we can just use an AST transformation
+revise_first(ex) = (revise(); return ex)
```

Comment:
Do you want this, or do you want `:($revise(); $ex)`?   i.e. don't you want to call `revise()` when the REPL actually executes the expression?   Or does it not matter?

Reply:
Or maybe something like `Expr(:toplevel, :($revise()), $ex)`, since e.g. `ex == :(import Foo)` can't work in a block.

Reply:
I think the only place where it matters is if the user edits a package providing another AST transform; with the version I have implemented, it will make updates in those packages immediately, including before they execute their transforms (for those packages that come later in the sequence than Revise). I *think* that's the behavior most people would want, so I'm inclined to leave it the way it is.

Reply:
I'm not sure if that use case is worth it for breaking the implicit interface that the AST transformations work by transforming the AST, rather than by side effects.    Suppose callers want to examine the transformed AST for other reasons than to execute it, for example.

Reply:
I wonder, though, if it would be better to move this to a `hooks` interface. But certainly I can live with the current interface, and I now agree with you that it's better to follow the implicit API.

Reply:
@KristofferC, will OhMyREPL be able to work within the AST-transform interface or do you need side-effects?

Reply:
OhMyREPal syntax highlights the text, it cannkt really do that with only AST transforms

Reply:
The idea being that OhMyREPL will steal the REPL but if nothing else needs it, all is good? Should I also push to `REPL.repl_ast_transforms` or will you grab from `active_repl_backend.ast_transforms`?

Reply:
> The idea being that OhMyREPL will steal the REPL but if nothing else needs it, all is good?

Yeah, pretty much. Not very nice of OhMyREPL but not sure how else I can do it.

Do I even have to do anything manually? The only thing OhMyREPL does is that it injects its own keybindings into the REPL but then it just calls the standard `on_done` methods. So hopefully that should just work?